src/neuroconv/nwbconverter.py

```diff
@@ -203,6 +204,14 @@ def run_conversion(
             Similar to source_data, a dictionary containing keywords for each interface for which non-default
             conversion specification is requested.
         """
+
+        if nwbfile_path is None:
```

Comment:
And so too should remove the default in the signature?

Reply:
Shouldn't we do that once the deadline is met?

Reply:
Oh, I see, we never actually warned about this level only the interface level

Nevermind